protobuf-definitions/make.bat

```diff
@@ -22,7 +22,7 @@ mkdir -p $DST_DIR_P/$PYTHON_PACKAGE
 # generate proto objects in python and C#
 
 protoc --proto_path=proto --csharp_out=$DST_DIR_C $SRC_DIR/*.proto
-protoc --proto_path=proto --python_out=$DST_DIR_P $SRC_DIR/*.proto 
+protoc --proto_path=proto --python_out=$DST_DIR_P --mypy_out=$DST_DIR_P $SRC_DIR/*.proto
```

Comment:
Have you tested this?

Reply:
Yes, that was how I generated the `.pyi` files.